### PR TITLE
Merge ID sets - add print and raise an exception

### PR DESCRIPTION
**Pull request by ChanochShayner**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: Fix names in merge ID sets command.

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by ChanochShayner** [comment]
test

File: demisto_sdk/__main__.py [hunk 1]
```diff
@@ -817,11 +817,15 @@ def merge_id_sets_command(**kwargs):
...
     output = kwargs['output']
 
-    merge_id_sets_from_files(
+    _, duplicates = merge_id_sets_from_files(
         first_id_set_path=first,
         second_id_set_path=second,
         output_id_set_path=output
     )
+    if duplicates:
+        print_warning(f'Failed to merge ID sets: {first} with {second}, '
```
Comment (guykeller) [inline demisto_sdk/__main__.py:826]: Why not `print_error`?

**Review by guykeller — APPROVED** [review]
Please fix or explain my one comment, then you're good to go

File: demisto_sdk/__main__.py (hunk 1 above)
Comment (ChanochShayner) [reply demisto_sdk/__main__.py]: done

**Pull request merged**